scripts/deploy_staging.ts

```diff
@@ -173,7 +176,22 @@ async function main() {
   await topUp.setUpkeepId(upkeepId);
   console.log("Upkeep ID set to DssVestTopUp contract");
 
-  console.log("TODO: Create a Uniswap pool for Test token and LINK");
+  const uniswapFactoryContract: Contract = new ethers.Contract(
+    UNISWAP_V3_FACTORY_ADDRESS,
+    abi
+  );
+
+  const createUniswapFactoryTx = await uniswapFactoryContract
+    .connect(admin)
+    .createPool(paymentToken.address, STAGING_LINK_TOKEN, 500);
```

Comment:
Which parameter is this `500` value for?

Reply:
This is for the pool fee, maybe I should move this to a constant so its more explicit?